modules/test_infrastructure/variables.tf

```diff
@@ -47,6 +47,9 @@ variable "vnets" {
                                 [VNET module documentation](../vnet/README.md#network_security_groups).
   - `route_tables`            - (`map`, optional) map of Route Tables to create, for details see
                                 [VNET module documentation](../vnet/README.md#route_tables).
+  - `local_peer_config`       - (`map`, optional) a map that contains local peer configuration parameters. This value allows to set `allow_virtual_network_access`, `allow_forwarded_traffic`, `allow_gateway_transit` and `use_remote_gateways` parameters on the local VNet peering. 
```

Comment:
Small remark to variables description, break the line when it exceeds column 130, similarly to other properties. This also applies to the description changes in the examples. Please fix that before merging, thanks!

Reply:
Thanks, I made an adjustment to the description.